enterprise/tasks.py

```diff
@@ -182,3 +207,133 @@ def send_sso_configured_email(
+    enterprise_customer = get_enterprise_customer(enterprise_customer_uuid)
+    group_membership = enterprise_group_membership_model().objects.filter(
+        uuid=group_membership_uuid
+    ).values().last()
+    braze_client_instance = BrazeAPIClient()
+    enterprise_catalog_client = EnterpriseCatalogApiClient()
+    braze_trigger_properties = {}
+    contact_email = enterprise_customer.contact_email
+    enterprise_customer_name = enterprise_customer.name
+    braze_trigger_properties['contact_admin_link'] = braze_client_instance.generate_mailto_link(contact_email)
+    braze_trigger_properties['enterprise_customer_name'] = enterprise_customer_name
+    braze_trigger_properties[
+        'catalog_content_count'
+    ] = enterprise_catalog_client.get_catalog_content_count(catalog_uuid)
+    braze_trigger_properties['budget_end_date'] = budget_expiration
+    if group_membership['enterprise_customer_user_id'] is None:
+        pending_enterprise_customer_user = pending_enterprise_customer_user_model().objects.get(
+            id=group_membership['pending_enterprise_customer_user_id'])
+        learner_email = pending_enterprise_customer_user.user_email
+        recipient = braze_client_instance.create_recipient_no_external_id(
+            learner_email,
+        )
+        # We need an alias record to exist in Braze before
+        # sending to any previously-unidentified users.
+        braze_client_instance.create_braze_alias(
+            [learner_email],
+            ENTERPRISE_BRAZE_ALIAS_LABEL,
+        )
+    else:
+        enterprise_customer_user = enterprise_customer_user_model().objects.get(
+            user_id=group_membership['enterprise_customer_user_id'])
+        learner_email = enterprise_customer_user.user_email
+        recipient = braze_client_instance.create_recipient(
+            user_email=learner_email,
+            lms_user_id=enterprise_customer_user.user_id,
+        )
+
+    try:
+        braze_client_instance.send_campaign_message(
+            settings.BRAZE_GROUPS_INVITATION_EMAIL_CAMPAIGN_ID,
+            recipients=[recipient],
+            trigger_properties=braze_trigger_properties,
+        )
+    except BrazeClientError as exc:
+        message = (
+            "Groups learner invitation email could not be sent "
+            f"to {learner_email} for enterprise {enterprise_customer_name}."
+        )
+        LOGGER.exception(message)
+        raise exc
+
+
+@shared_task
+@set_code_owner_attribute
+def send_group_membership_removal_notification(enterprise_customer_uuid, group_membership_uuid):
+    """
+    Send braze email notification when learner is removed from a group.
+
+    Arguments:
+        * enterprise_customer_uuid (string)
+        * group_membership_uuid (string)
+    """
+    enterprise_customer = get_enterprise_customer(enterprise_customer_uuid)
+    group_membership = enterprise_group_membership_model().objects.filter(
+        uuid=group_membership_uuid
+    ).values().last()
```

Comment:
...and same idea here about using related records via the `ForiegnKey` relationship.